{ 'fall_position_front_2': { 'head': [ { 'duration': 1.0,
                                         'name': u'fall_position_front',
                                         'positions': [0.0, 0.0012],
                                         'starttime': 0.0}],
                             'left_arm': [ { 'duration': 1.0,
                                             'name': u'fall_position_front_2',
                                             'positions': [ -2.16,
                                                            -1.43,
                                                            0.085,
                                                            0.036,
                                                            0.9311,
                                                            3.2208,
                                                            0.02],
                                             'starttime': 0.0}],
                             'left_leg': [ { 'duration': 1.0,
                                             'name': u'fall_position_front',
                                             'positions': [ 0.0063,
                                                            -0.0365,
                                                            -0.8302,
                                                            1.4101,
                                                            0.6402,
                                                            -0.0324],
                                             'starttime': 0.0}],
                             'right_arm': [ { 'duration': 1.0,
                                              'name': u'fall_position_front_2',
                                              'positions': [ 2.16,
                                                             1.43,
                                                             -0.085,
                                                             -0.036,
                                                             -0.9311,
                                                             -3.2208,
                                                             -0.02],
                                              'starttime': 0.0}],
                             'right_leg': [ { 'duration': 1.0,
                                              'name': u'fall_position_front',
                                              'positions': [ -0.0063,
                                                             0.0365,
                                                             0.8302,
                                                             -1.4101,
                                                             -0.6402,
                                                             0.0324],
                                              'starttime': 0.0}],
                             'torso': [ { 'duration': 1.0,
                                          'name': u'fall_position_front',
                                          'positions': [-0.0003, -0.0067],
                                          'starttime': 0.0}]}}
